tests/expressions/models.py

```diff
@@ -102,3 +102,15 @@ class UUIDPK(models.Model):
 class UUID(models.Model):
     uuid = models.UUIDField(null=True)
     uuid_fk = models.ForeignKey(UUIDPK, models.CASCADE, null=True)
+
+
+class Item(models.Model):
+    pass
+
+
+class ItemVersion(models.Model):
+    item = models.ForeignKey(Item, related_name="versions", on_delete=models.CASCADE)
+    version_number = models.IntegerField(default=0)
+
+    class Meta:
+        unique_together = ("item", "version_number")
```

Comment:
Can we reuse existing models?

As far as I'm aware `unique_together` is not required :thinking:

Reply:
Updated to have the one test with existing models - I think it covers the ticket requirement :+1: